src/dlgConnectionProfiles.cpp

```diff
@@ -1421,20 +1422,31 @@ void dlgConnectionProfiles::slot_copy_profile()
         return;
     }
 
-    copyFolder(mudlet::getMudletPath(mudlet::profileHomePath, oldname), mudlet::getMudletPath(mudlet::profileHomePath, profile_name));
-    mProfileList << profile_name;
-    slot_item_clicked(pItem);
-    // Clear the Discord optin on the copied profile - just because the source
-    // one may have had it enabled does not mean we can assume the new one would
-    // want it set:
-    discord_optin_checkBox->setChecked(false);
+    QApplication::setOverrideCursor(Qt::BusyCursor);
+    mpCopyProfile->setText(tr("Copying..."));
```

Comment:
You could enhance the UI effect by making it a "checkable" button and only unchecking it (using the method that does NOT cause the triggered signal to be raised) when the copy is completed and you are re-enabling it again...

Reply:
Hmm, I don't think that will look better.

Reply:
Well I've done it elsewhere, it means that the button clicks down - and as it gets disabled - it stays down (and is un-unclickable) until the job is done - then it springs back up again and lights up as being available...

Reply:
OK, but I don't think that would be a better experience - see the video in https://github.com/Mudlet/Mudlet/pull/5056#discussion_r846006536, the current one works well.